lib/Client/Transports/TransportHTTPPROTO.php

```diff
@@ -0,0 +1,59 @@
+<?php
+namespace LightStepBase\Client\Transports;
+
+class TransportHTTPPROTO {
+
+    protected $_host = '';
+    protected $_port = 0;
+    protected $_verbose = 0;
+
+    public function ensureConnection($options) {
+        $this->_verbose = $options['verbose'];
+
+        $this->_host = $options['collector_host'];
+        $this->_port = $options['collector_port'];
+
+        // The prefixed protocol is only needed for secure connections
+        if ($options['collector_secure'] == True) {
+            $this->_host = "ssl://" . $this->_host;
```

Comment:
Probably out of scope, but shouldn't we be using `tls` instead of `ssl`?

Reply:
https://github.com/lightstep/lightstep-tracer-php/issues/17